python/shark_turbine/dynamo/importer.py

```diff
@@ -373,6 +373,13 @@ def _import_torch_op_overload(
         if schema.overload_name != "":
             mlir_op_name += f".{schema.overload_name}"
 
+        # Intervening to use Scalar ops due to incorrect ops from AOT-autograd with scalar arguments.
+        if mlir_op_name in TENSOR_SCALAR_OP_CONVERTER:
```

Comment:
Nit: for performance sensitive code, I usually go to the trouble of not doing two lookups: out it in a temporary with .get and then condition in not none.

Reply:
Thanks for the feedback! Super helpful for my growth :)